client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -303,6 +338,10 @@ const formatStatsData = ( site: Site ) => {
 		type: 'stats',
 		data: site.site_stats,
 	};
+	const weeklyStats = site.site_stats.views.total;
+	statsData.status = site.site_stats.views.trend;
+	//statsData.data.views.total = weeklyStats;
+	statsData.value = weeklyStats;
```

Comment:
I'm still having trouble here.

Line 344 works, but has an error message about the `value`

I'd expect 343 to work, but something must be wrong with the syntax.

Any help is appreciated!

Reply:
Reworked this whole area.